packages/manager/apps/web/client/app/dns-zone/history/dashboard/view/view.html

```diff
@@ -0,0 +1,15 @@
+<oui-modal
+    data-heading="{{:: 'dashboard_history_popup_title' | translate }}"
+    data-primary-action="$ctrl.goBack()"
+    data-primary-label="{{:: 'dashboard_history_close_popup' | translate }}"
+    data-secondary-disabled="true"
+    data-on-dismiss="$ctrl.goBack()"
+    data-loading="$ctrl.loadingDnsZoneData"
+>
+    <div
+        style="white-space: pre-wrap;"
+        ng-if="!$ctrl.loadingDnsZoneData"
+        class="modal-scroll-auto"
+        ng-bind="$ctrl.dnsZoneData"
+    ></div>
```

Comment:
```suggestion
    <pre
        data-ng-bind="$ctrl.dnsZoneData"
    ></pre>
```
I think a `<pre>` is more adapted for your case.